guides/hack/05-functions/05-format-strings.md

```diff
@@ -36,3 +35,19 @@ function use_it(): void {
 `HH\FormatString<PlainSprintf>` will check that you've used the right
 number of arguments. `HH\FormatString<Str\SprintfFormat>` will also
 check that arguments match the type in the format string.
+
+## Format Specifiers
+See `PlainSprintf` for in-depth information on all format specifiers. 
+
+| Specifier | Expected Input | Expected Output                                                                                                     |
+|-----------|----------------|---------------------------------------------------------------------------------------------------------------------|
+| b         | int            | Binary (`63` => `111111`)                                                                                           |
+| c         | int            | ASCII character (`63` => `?`)                                                                                       |
+| d, u      | int            | Signed int (d: `-1` => `-1`) \| Unsigned int (u: `-1` => `18446744073709551615`)                                    |
```

Comment:
Could we split `d` and `u` onto separate lines, since they give different output?